Philippines/aral.md

```diff
@@ -0,0 +1 @@
+Kung gusto mo pa mag-aral. eh di bumalik ka sa Udacity at kunin lahat ng kursong gusto mo. Good luck sa iyo! 
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.